.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Android APK Build
+
+on:
+  push:
+    branches:
+      - 'release/**' # Only trigger on branches starting with 'release/'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout the code
+      uses: actions/checkout@v2
+
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
```

Comment:
Consider upgrading to ``actions/setup-java@v3`` for better performance, caching, and security.